client/src/components/OtherUserAvatar/OtherUserAvatar.tsx

```diff
@@ -0,0 +1,36 @@
+import { Box, Badge, Avatar } from '@material-ui/core';
+import useStyles from './useStyles';
+
+interface Props {
+  username: string;
+  profileImg?: string;
+  small?: boolean;
+  isOnline?: boolean;
+  customClasses?: string;
+}
+
+const OtherUserAvatar = ({ username, profileImg, small, isOnline, customClasses }: Props): JSX.Element => {
+  const classes = useStyles();
+
+  const badgeStyles = (isOnline: boolean | undefined): string => {
+    if (isOnline) {
+      return `${small ? classes.onlineBadge : classes.onlineActiveChatBadge}`;
+    } else {
+      return '';
+    }
+  };
+
+  return (
+    <Box className={`${classes.avatar} ${customClasses}`}>
+      <Badge
+        classes={{ badge: badgeStyles(isOnline) }}
```

Comment:
Optional: You can use a light weight package like 'clsx' to conditionally apply styles to MUI components :)

Eg: 
```
className={clsx({   
          [classes.onlineBadg]: isOnline && small,             
          [classes.onlineActiveChatBadge]: isOnline && !small,           
        })}
```

Ref: https://stackoverflow.com/questions/57557271/how-to-use-clsx-in-react

Reply:
Applied.
That's exactly what I needed! Thank you 🙂